olp-cpp-sdk-dataservice-read/src/repositories/PartitionsRepository.cpp

```diff
@@ -288,6 +350,82 @@ model::Partition PartitionsRepository::PartitionFromSubQuad(
   return ret;
 }
 
+PartitionsResponse PartitionsRepository::GetAggregatedTile(
+    const client::HRN& catalog, const std::string& layer,
+    client::CancellationContext context, const TileRequest& request,
+    boost::optional<int64_t> version, const client::OlpClientSettings& settings,
+    bool aggregated) {
```

Comment:
As i said we dont need the `bool aggregated` parameter. Just hard code to true in FindPartition().